index.html

```diff
@@ -1,37 +1,115 @@
+          <li>bootstrap</li>
+          <li>Ruby</li>
         </ul>
+        <a href="#" class="story button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
       </div>
-    </section>
```

Comment:
- [ ] Kindly add the about skills, frameworks and languages after this section, as this is part of this milestone 
![Screenshot_79](https://user-images.githubusercontent.com/69638013/165212049-a00fd474-c855-4c8d-b6d9-b07efcdd3043.png)

Reply:
- [ ] Please add the above section recommended by the previous reviewer, it is a mandatory section and it should be added to your project in order for it to be approved 👍 . It should look like this 
![image](https://user-images.githubusercontent.com/45460192/165361509-a66ebeec-f2bb-4e71-b701-f7a0f16c17a6.png)